### PR TITLE
New Jersey!

### PR DESCRIPTION
**Pull request by tannewt**

### CODE REVIEW

File: states/new_jersey/elections.toml [hunk 1]
```diff
@@ -3,7 +3,140 @@ date = 2020-07-07
...
+original_date = 2020-09-29
+
+[20200929.registration]
+received_by = 2020-09-08
+in_person_by = 2020-09-08
+sources = ["https://www.state.nj.us/state/elections/voter-registration.shtml",
+           "https://www.state.nj.us/state/elections/assets/pdf/chrons/2020-chron-08-september-school-board.pdf"]
+
+[20200929.absentee.application]
+received_by = 2020-09-29
```
Comment (askpatrickw) [inline states/new_jersey/elections.toml]: I think received_by ~~and possibly in_person_by~~ should be 2020-09-22

From the sources: 
September 22 Deadline to apply for a Mail-In Ballot by Mail for September Special School Board Elections(not less than 7 days prior to election)N.J.S.A. 19:63-3 

(In_person_by is fine)

File: states/new_jersey/elections.toml [hunk 2]
```diff
@@ -3,7 +3,140 @@ date = 2020-07-07
...
+           "https://lis.njleg.state.nj.us/nxt/gateway.dll/statutes%2F1%2F19539%2F19749"]
+
+[20200929]
+date = 2020-09-29
+name = "September Special School Board"
+original_date = 2020-09-29
+
+[20200929.registration]
+received_by = 2020-09-08
+in_person_by = 2020-09-08
```
Comment (askpatrickw) [inline states/new_jersey/elections.toml:43]: I didn't see drop-off and mail-in called out separately in the sources... Is there an additional source?

File: states/new_jersey/elections.toml [hunk 3]
```diff
@@ -3,7 +3,140 @@ date = 2020-07-07
...
+
+[20200929.absentee.application]
+received_by = 2020-09-29
+in_person_by = 2020-09-28T15:00:00
+sources = [
+    "https://www.state.nj.us/state/elections/assets/pdf/chrons/2020-chron-08-september-school-board.pdf"
+]
+
+[20200929.absentee.military_overseas.application]
+online_by = 2020-09-25
```
Comment (askpatrickw) [inline states/new_jersey/elections.toml:55]: I was also thinking we might need online_by for .registration its frequently available online as well. I don't recall it being different than received_by,  but we may want to always include online_by and received_by if there are two methods and the same date... also feels like extra work, but it also more accurate. Thoughts ?

File: states/new_jersey/elections.toml [hunk 4]
```diff
@@ -3,7 +3,140 @@ date = 2020-07-07
...
+
+[20201103.absentee.application]
+received_by = 2020-10-27
+in_person_by = 2020-11-02T15:00:00
+sources = [
+    "https://www.state.nj.us/state/elections/assets/pdf/chrons/2020-chron-general-election-040820.pdf"
+]
+
+[20201103.absentee.military_overseas.application]
+online_by = "2020-07-03"
```
Comment (askpatrickw) [inline states/new_jersey/elections.toml]: online_by = 2020-10-30

From Sources: 
October 30  Deadline for Application to Receive General Election Mail-In Ballots by Electronic Means for Qualified Overseas Civilian and Military Voters(4th day before election)

**Review by askpatrickw — CHANGES_REQUESTED** [review]
Look pretty good! just some inline comments corrections I noted.

File: states/new_jersey/elections.toml (hunk 3 above)
Comment (tannewt) [reply states/new_jersey/elections.toml:55]: Oh ya, that would be good.

File: states/new_jersey/elections.toml (hunk 2 above)
Comment (tannewt) [reply states/new_jersey/elections.toml:43]: I don't think they are but both are possible so they should be listed. `Mail or deliver the Voter Registration Application and/or Party Affiliation Form to the County Commissioner of Registration or Superintendent of Elections for your county. `

File: states/new_jersey/elections.toml (hunk 3 above)
Comment (tannewt) [reply states/new_jersey/elections.toml:55]: When multiple methods are doable they should have separate entries, even if the dates are the same. That way we'll generate reminders for each.

**Review by askpatrickw — APPROVED** [review]

**Comment by askpatrickw** [comment]
And now I get to say Thank you to you ! ;-)

**Pull request merged**